pkg/fixtures/writer_s3.go

```diff
@@ -0,0 +1,103 @@
+package fixtures
+
+import (
+	"context"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"github.com/aws/aws-sdk-go/aws"
+	"os"
+	"path/filepath"
+)
+
+type BlobFixturesSettings struct {
+	BasePath   string
+	ConfigName string
+}
+
+type s3FixtureWriter struct {
+	logger      mon.Logger
+	store       blob.Store
+	batchRunner *blob.BatchRunner
+
+	settings *BlobFixturesSettings
+	purger   *s3Purger
+}
+
+func S3FixtureWriterFactory(settings *BlobFixturesSettings) FixtureWriterFactory {
+	return func(config cfg.Config, logger mon.Logger) (FixtureWriter, error) {
+		store := blob.NewStore(config, logger, settings.ConfigName)
+		br := blob.NewBatchRunner(config, logger)
+		purger := newS3Purger(config, logger, settings)
+
+		return NewS3FixtureWriterWithInterfaces(logger, store, br, settings, purger), nil
+	}
+}
+
+func NewS3FixtureWriterWithInterfaces(logger mon.Logger, store blob.Store, batchRunner *blob.BatchRunner, settings *BlobFixturesSettings, purger *s3Purger) FixtureWriter {
+	return &s3FixtureWriter{
+		logger:      logger,
+		store:       store,
+		settings:    settings,
+		batchRunner: batchRunner,
+		purger:      purger,
+	}
+}
+
+func (s *s3FixtureWriter) Purge() error {
+	return s.purger.purge()
+}
+
+func (s *s3FixtureWriter) Write(*FixtureSet) error {
```

Comment:
not sure if this is a personal flavour, but i believe we don't have a rule for that yet.
I like writing it like so:
```suggestion
func (s *s3FixtureWriter) Write(_ *FixtureSet) error {
```
@j4k4 any feelings about this?

Reply:
what should I answer to this? x)

Reply:
This might be the one discussion where i am actually waiting for a reaction for @j4k4. However not a blocker, if we don't get a response. It's not a no-no, but rather an "i'm not sure how to handle this case -> raising to project owner" :wink:

Reply:
Yeah, we don't have a rule for this yet..  
I tend to write it like `func (s *s3FixtureWriter) Write(_ *FixtureSet) error {` as well, but i have no hard feelings about it..  
I guess most places have the underscore, so maybe we should stick to it?

Reply:
agreed, thanks :-)

Reply:
Applied there.